### PR TITLE
Serving models - updates

### PR DESCRIPTION
**Pull request by omesser**
- ~update mlem serve command~ - my bad, was using older mlem
- add curl CLI example
- and update verbiage a bit

### CODE REVIEW

**Comment by aguschin** [comment]
@omesser, please note that PR from forks doesn't allow to deploy this to preview envs https://github.com/iterative/mlem.ai/pull/211#issuecomment-1302617352

File: content/docs/get-started/serving.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ serving models via REST API is quite common.
 To start up a FastAPI server run:
 
 ```cli
-$ mlem serve fastapi --model models/rf
+$ mlem serve models/rf fastapi
```
Comment (aguschin) [inline content/docs/get-started/serving.md:12]: I believe you're using MLEM < 0.3.0.

Comment (aguschin) [reply content/docs/get-started/serving.md:12]:
```suggestion
$ mlem serve fastapi --model models/rf
```

File: content/docs/get-started/serving.md [hunk 2]
```diff
@@ -31,10 +34,37 @@ That's it! You can check out the other available server implementations in
...
+check out OpenAPI spec and query examples.
+
+Or, alternatively, launch requests at the server from a different terminal,
+Using `curl` or `httpie`. For example:
 
-MLEM can help you with not only serving, but also querying the model. Using MLEM
-API or CLI you can send requests to your model without implementing the "client"
-part. Learn more about this in [User Guide](/doc/user-guide/serving/).
+```cli
+❯ curl -X 'POST' \
```
Comment (aguschin) [inline content/docs/get-started/serving.md:45]:
```suggestion
$ curl -X 'POST' \
```

we use `$` in docs

**Comment by aguschin** [comment]
Errr, http://example-mlem-get-started-app.herokuapp.com doesn't loads for me. I see some recent requests to it though. Looks like heroku can't handle the load? 🤔

File: content/docs/get-started/serving.md (hunk 1 above)
Comment (omesser) [reply content/docs/get-started/serving.md:12]: @aguschin you're right.
Not sure why 🤦  my bad

**Comment by omesser** [comment]
> @omesser, please note that PR from forks doesn't allow to deploy this to preview envs [#211 (comment)](https://github.com/iterative/mlem.ai/pull/211#issuecomment-1302617352)
> 
> Next time please just create a branch with changes in this exact repo :)

continuing here https://github.com/iterative/mlem.ai/pull/217 - from a branch pushed to this fork (upstream).
But this repo is a public repo - it should work with contributions (=prs from forks) - fully 😄 I hope we can fix this soon

**Comment by jorgeorpinel** [comment]
> this repo is a public repo - it should work with contributions (=prs from forks) - fully

This is a discussion for @iterative/websites I think, but some of the features (example Restyled) specifically cannot be trusted to work on forked PRs due to security concerns. Probably the same with review apps: this would give any random user the ability to deploy things to our test infrastructure.

**Pull request closed without merging**